### PR TITLE
[ENT-8956] - fix: missing href on xml download button

### PR DESCRIPTION
**Pull request by hamzawaleed01**
# For all changes

- [ ] Ensure adequate tests are in place (or reviewed existing tests cover changes)

# Only if submitting a visual change

- [ ] Ensure to attach screenshots
- [ ] Ensure to have UX team confirm screenshots

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -57,3 +57,4 @@ PLOTLY_SERVER_URL='http://localhost:8050'
 AUTH0_SELF_SERVICE_INTEGRATION='true'
 MFE_CONFIG_API_URL='http://localhost:18000/api/mfe_config/v1'
 DEMO_ENTEPRISE_UUID='set a valid enterprise uuid'
+EDX_ACCESS_URL='http://localhost:1991'
```
Comment (hamzawaleed01) [inline .env.development:60]: @sameenfatima78 @zamanafzal we don't have this service in local env. What should be the default url here then? 🤔

Comment (sameenfatima78) [reply .env.development:60]: Wondering which service is this pointing to? Does it have a repo?

Comment (hamzawaleed01) [reply .env.development:60]: nope, it's private - managed by auth team.

Comment (hamzawaleed01) [reply .env.development:60]: I've just added the base url of its own (frontend-app-admin-portal)

Comment (hamzawaleed01) [reply .env.development]: Setting it to testing env of auth service which is `http://2u-guid-dev.us.auth0.com/`

File: .env.development [hunk 2]
```diff
@@ -57,3 +57,4 @@ PLOTLY_SERVER_URL='http://localhost:8050'
 AUTH0_SELF_SERVICE_INTEGRATION='true'
 MFE_CONFIG_API_URL='http://localhost:18000/api/mfe_config/v1'
 DEMO_ENTEPRISE_UUID='set a valid enterprise uuid'
+EDX_ACCESS_URL='http://2u-guid-dev.us.auth0.com'
```
Comment (zamanafzal) [inline .env.development:60]: I guess we should set it empty here considering its opensource and `http://2u-guid-dev.us.auth0.com` will not be available to everyone.

**Review by zamanafzal — APPROVED** [review]

**Pull request merged**